app/tests/utils/test_run_execution.py

```diff
@@ -85,12 +87,15 @@ def create_random_test_run_execution_archived(
 
 
 def create_random_test_run_execution(
-    db: Session, selected_tests: Optional[TestSelection] = {}, **kwargs: Any
+    db: Session,
+    selected_tests: Optional[TestSelection] = {},
+    pics: Optional[PICS] = PICS(),
```

Comment:
Maybe if you add this default value you wouldn't even need to update the other methods in this file?
Unless you prefer to keep the default value as an empty PICS, which is valid as well.
```suggestion
    pics: Optional[PICS] = create_random_pics(),
```

Reply:
You are right @ccruzagralopes, I'd rather keep PICS() just to keep empty as default value.